htdocs/install.sh

```diff
@@ -361,15 +361,15 @@ elif [ $ID == "opensuse" ] || [ $ID == "suse" ]; then
 	cat /dev/null | $SUDO rpm --import /tmp/zt-gpg-key
 
 	echo
-	echo '*** Installing zerotier-one package...'
```

Comment:
This change reverts spelling correction

Reply:
looks like you edited htdocs/install.sh.  Only  /install.sh.in should be edited

Reply:
Ah yeah, I wasn't sure of the relationship between those and see the build script now. I reverted the changes to htdocs/install.sh

Reply:
@glimberg Does that imply that /htdocs/install.sh should be a linked file to /install.sh.in and excluded via git config?

Reply:
no not at all.  `htdocs/install.sh` is a copy of `install.sh.in` that includes the GPG signature of the content